ydb/library/yql/sql/v1/SQLv1Antlr4.g.in

```diff
@@ -258,7 +258,7 @@ literal_value:
   | real
   | STRING_VALUE
   | BLOB // it's unused right now
-  | NULL
+  | NULL@GRAMMAR_RESERVED_SUFFIX@
```

Comment:
Disadvantage is that it makes grammar way less readable. If it is not acceptable, then maybe there is some way to configure this grammar using some regex based replacement?